rust/operator-binary/src/controller/build/node_config.rs

```diff
@@ -288,13 +400,14 @@ mod tests {
         kvp::LabelValue,
         product_logging::spec::AutomaticContainerLogConfig,
         role_utils::GenericRoleConfig,
+        shared::time::Duration,
     };
     use uuid::uuid;
 
     use super::*;
     use crate::{
         controller::{ValidatedLogging, ValidatedOpenSearchConfig},
-        crd::NodeRoles,
+        crd::{NodeRoles, v1alpha1::OpenSearchTls},
```

Comment:
It is common practice at Stackable to only import the versioned module and explicitly reference it everywhere.

```suggestion
        crd::{NodeRoles, v1alpha1},
```